Integrations/integration-Twilio.yml

```diff
@@ -0,0 +1,101 @@
+description: Send SMS notifications
+detaileddescription: "To use Twilio:\n1.\tRegister to Twilio at: https://www.twilio.com/try-twilio\n2.\tAfter
+  completing the registration go to: https://www.twilio.com/user/account/messaging/getting-started\n3.\tSelect
+  “show API credentials”\n4.\tCopy the Account SID and AUTH TOKEN\n5.\tIf you do not
+  have a sending number, Go to: https://www.twilio.com/user/account/phone-numbers/getting-started\nSelect:
+  “get your first Twilio phone number”. Follow the instructions and copy the chosen
+  number.\n6.\tOnce complete use the “!sms-send” command to send text messages"
+configuration:
+- display: Server URL (e.g. https://192.168.0.1)
+  name: server
+  defaultvalue: https://api.twilio.com/2010-04-01/Accounts
+  type: 0
+  required: true
+- display: Account SID
+  name: sid
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Auth token
+  name: token
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Default sending number
+  name: sender
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Do not validate server certificate (insecure)
+  name: insecure
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: ""
+  type: 8
+  required: false
+script:
+  script: |2+
+
+
+
+    var sendRequest = function(url, body, headers) {
+            url.replace(/[\/]+$/, '');
+            var res = http(
+                url,
+                {
+                    Method: 'POST',
+                    Body: body,
+                    Headers: headers,
+                    Username: params.sid,
+                    Password: params.token
+                },
+                params.insecure,
+                params.proxy
+            );
+            if (res.StatusCode < 200 || res.StatusCode >=300) {
+                throw 'Request Failed.\nStatus code: ' + res.StatusCode + '.\nBody: ' + JSON.stringify(res) + '.';
+            }
+            return res.Body;
+        };
+
+
+
+
+    var headers = {'Content-Type': ['application/x-www-form-urlencoded']};
+    var url = params.server.replace(/[\/]+$/, '')+'/'+params.sid+'/Messages.json' ;
+    // The command input arg holds the command sent from the user.
+    switch (command) {
+        // This is the call made when pressing the integration test button.
+        case 'test-module':
+            var body = "From=%2B"+params.sender.replace(/^\+/, '')+"&To=%2B"+params.sender.replace(/^\+/, '')+"&Body="+args.body;
```

Comment:
you should url encode the body using encodeURIComponent(args.body) before sending it